<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addGsm</class>
 <widget class="QWidget" name="addGsm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить путевой лист</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>2</y>
     <width>391</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Новый путевой лист</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Дата:...............................................</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>№ Путевого листа:..............................</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Показания спидометра при выезде,км:.....</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Пройдено по городу,км:........................</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Пройдено по трассе,км:........................</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Показания спидометра при возвращении,км</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDateEdit" name="lineDate"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNumber"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineStartMileage"/>
       </item>
       <item>
        <widget class="QLineEdit" name="linePoGorodu"/>
       </item>
       <item>
        <widget class="QLineEdit" name="linePoTrasse"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineProbegReturned"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>231</y>
      <width>225</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Остаток ГСМ при выезде,л:....................</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Расход топлива по городу(факт),л:.........</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Расход топлива по трассе(факт),л:.........</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Выдано ГСМ,л:...................................</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Остаток ГСМ при возвращении,л:...........</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Расход топлива по городу(норма),л:........</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Расход топлива по трассе(норма),л:.........</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>229</y>
      <width>135</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineStartGsm"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linePoGoroduGsm"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linePoTrasseGsm"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineGsmGiven"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineGsmBack"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linePoGoroduGsmNorm"/>
     </item>
     <item>
      <widget class="QLineEdit" name="linePoTrasseGsmNorm"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>7</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
